cmd/launcher/svc_windows.go

```diff
@@ -190,3 +196,74 @@ func (w *winSvc) Execute(args []string, r <-chan svc.ChangeRequest, changes chan
 		}
 	}
 }
+
+const (
```

Comment:
I'd toss all this into `cmd/launcher/svc_checker_windows.go` or something

Reply:
Will do!